.github/workflows/authorized-tag-creators.txt

```diff
@@ -0,0 +1 @@
+manaswinidas
```

Comment:
This is a list we should expand if it's dynamically read. 

Managers at a min (maybe a good start?), advisors should all be here (would make the most sense given their current responsibilities)... I don't have a problem with you being here, but I think if this is an actual grant list, we should definitely make sure it is prefilled with a good set of people so we are not locked out of anything.

Reply:
I tested by pushing the changes to my fork and personal Quay repository with my secrets added to the repository. 

Ok, adding the advisors and managers. I didn't add you and Lucas coz you both already have access 😛

Reply:
Added myself coz I tested with the same.

Reply:
Push the changes?

Reply:
@manaswinidas can you take a screenshot of what you're seeing? I'm not seeing an update here. I think you're not pushing 🤔

Reply:
just added advisors and managers to the text file.. don't mind my previous comment I thought I wrote something weird

Reply:
About testing: here's a failed workflow run when I didn't add myself to the file: https://github.com/manaswinidas/odh-dashboard/actions/runs/12182131959/job/33980387355 and this is passing workflow run when I added myself to the file(this created a tag in my personal Quay repo on successful run): https://github.com/manaswinidas/odh-dashboard/actions/runs/12182092368/job/33980257364. Not sure whether you can access these links.

Reply:
And this is the workflow that I used there: https://github.com/manaswinidas/odh-dashboard/blob/main/.github/workflows/create-tag-release.yml